fcomm/tests/proof_tests.rs

```diff
@@ -304,10 +304,8 @@ fn test_create_open_and_verify_complicated_higher_order_functional_commitment1()
     );
 }
 
-// #[test]
-// #[ignore]
-// FIXME: This fails to verify, which seems to be a circuit bug.
-#[allow(dead_code)]
+#[test]
+#[ignore]
```

Comment:
Although the corresponding box was checked as done in #71, this was still failing and commented out. It was incidentally fixed in this work.